### DISCUSSION TITLE
Purge files older than X days

### DISCUSSION THREAD

**Discussion by flick1999** [pymedusa/Medusa#5758] [Ideas & Feature requests]
**Is your feature request related to a problem? Please describe.**
There are many daily shows, such as Jimmy Kimmel or The Daily Show, that I don't want/need to keep around forever, whether I watch them or not. These can build up over time and take up a lot of space.

**Describe the solution you'd like**
I'd like a Purge option added to a show's Advanced page. It would look something like:
    Purge Older Episodes? 
         Yes/No
    Purge By: 
          Age - Older than X days
          Season - Keep last X seasons
          Count - Keep last X episodes

**Describe alternatives you've considered**
1. I've looked into having Plex doing it but it only deletes the media files, not all of the associated Kodi files that I have Medusa create. Using Medusa would ensure it removes all associated files.
2. I currently use a shell script to delete files older than X days in the show folder, but that ends up removing all of the non-show related files, such as tvshow.nfo, and in some cases, when a show hasn't aired in awhile due to hiatus, removes the entire show folder! When that latter happens, Medusa isn't able to process new files when they come down until I manually recreate the folder.

**Additional context**
None. Thank you for your consideration.

**Comment by a10kiloham** [comment]
I'd use this. +1

**Comment by flick1999** [comment]
I appreciate that, @a10kiloham. I think you need to +1 (aka thumbs up) the original post for the github robot to notice and log it. Thanks!

**Comment by a10kiloham** [comment]
👍
